apps/main/src/app/.well-known/walletconnect.txt/route.ts

```diff
@@ -1,8 +1,19 @@
 import { headers } from 'next/headers'
-import { getVercelDomainVerification } from '@ui-kit/features/connect-wallet/lib/utils/walletModules'
+
+const WALLET_CONNECT_ACCOUNT = `c3fe8dd8-93df-44af-803f-83798aa1d440`
+
+// for curve-dapp-git-feat-connect-modal-curvefi.vercel.app, other domains can be added the dashboard
+const VERCEL_DOMAIN_VERIFICATION = '0c282d3503877d627a6a75095ebdb60530b900d3b194d046f6d8da3ae19fd48e'
+// for curve.fi and staging.curve.fi
+const CURVE_DOMAIN_VERIFICATION = '3d76b3cd8cd754f34ac1c18ff25dc23ee9b80fc7f75800041335263b11f20b19'
+
+const getVercelDomainVerification = (host: string) =>
```

Comment:
I think I chose this name, but it's not really correct
```suggestion
/** Get the token used to verify the domain in the wallet connect dashboard */
const getDomainVerificationToken = (host: string) =>
```